lib/inferno/config/boot/ig_files.rb

```diff
@@ -0,0 +1,29 @@
+require 'fileutils'
+
+Inferno::Application.register_provider(:ig_files) do
+  prepare do
+    # This process should only run once, so skipping it on workers will start it
+    # only once from the "web" process
+    next if Sidekiq.server?
+
+    target_container.start :logging
```

Comment:
This means that logging has to start before this, right? Should we have a similar dependency that this has to run before the validator boot process? https://github.com/inferno-framework/inferno-core/blob/main/lib/inferno/config/boot/validator.rb

Reply:
Yes, good catch.